pkg/cli/admin/release/mirror.go

```diff
@@ -681,6 +681,10 @@ func (o *MirrorOptions) Run(ctx context.Context) error {
 		hasErrors := make(map[string]error)
 		maxPerIteration := 12
 
+		importMode := imagev1.ImportModeLegacy
+		if o.KeepManifestList {
+			importMode = imagev1.ImportModePreserveOriginal
```

Comment:
Sounds reasonable and `istag` seems more aligned with what is desired. What will be the impact on the mirrored payload?

Reply:
Hi, as in the reported logs in the jira ticket:

### without this change

ManifestList payload: it's 'unpacked', and we import the manifests for the same architecture as control plane of the cluster.
Manifest payload: no impact

### With this change

ManifestList payload: imported as ManifestList
Manifest payload: no impact

Reply:
However, PreserveOriginal can eventually become the default... I could change the object only when keep-manifest-list is set to make this change future-proof... wdyt?

Reply:
In my opinion, this way is more readable, instead of a hidden default. In any case, there will be a PR, when the default is changed and we can make change here too.